fedmsg/core.py

```diff
@@ -139,7 +139,8 @@ def __init__(self, **config):
                     # If we fail to bind or connect, there's probably another
                     # process already using that endpoint port.  Try the next
                     # one.
-                    pass
+                    self.log.debug(
```

Comment:
Technically this isn't a PEP-8 change or minor refactor, so although I'd definitely accept a change like this, I'd prefer it to be in a separate PR